### ISSUE TITLE
Other atsamx7x works in progress

### ISSUE THREAD

**Issue by ju6ge** [jamesmunns/same70-experiments#1]
Hey, i just saw that you are doing some work on the atsam architecture. A while back i did some work with rust on it as well. I never had the time to upstream the code to michfaltas repo. Because neither of us had time. But maybe it is useful to you ;)

https://github.com/ju6ge/atsamx7x-hal

**Comment by jamesmunns** [comment]
Oh excellent! I'm planning on focusing on Ethernet so far, so it looks like this will merge well with your work.

Thanks for the heads up!

**Comment by ju6ge** [comment]
You are welcome ;)

**Comment by jamesmunns** [comment]
If you don't mind, I'd like to keep this issue open so I don't forget about it! Feel free to unsubscribe if you'd like, but that "Issues (1)" on the repo view is helpful for me :)

**Comment by ju6ge** [comment]
fair enough :laughing: … if you have any question about my code feel free to ask ;)

**Comment by martinmortsell** [comment]
Commenting here, as requested on the RTIC-matrix.
